docs/conf/prague/2020/tickets.rst

```diff
@@ -28,19 +28,19 @@ We have a `waiting list <https://ti.to/writethedocs/write-the-docs-{{shortcode}}
 Ticket details
 ~~~~~~~~~~~~~~
 
-Each ticket includes:
+Write the Docs {{ name }} {{ year }} is a virtual conference. Each ticket includes:
 
-* Entry to all conference events and activities
-* Breakfast, snacks, and lunch on all event days (Sunday-Tuesday)
-* Welcome Reception and Social Event with light snacks and drinks
-* Wifi throughout the event
-* Meeting lots of fantastic people in a spacious, inviting venue
+* Live streaming of all talks
+* Q&A with speakers after each talk (may not be available for all speakers)
+* Access to the conference chat with all other attendees, speakers and sponsors
+* Access to the writing day
+* The virtual job fair
 
 Refund Policy
 -------------
 
 Refunds will be offered with a 10% processing fee until 2 weeks before the conference.
-We can't do refunds after that date because all of our catering, badge, & swag orders will have been placed.
+We can't do refunds after that date.
```

Comment:
```suggestion
Refunds after this date are not possible.
```

Reply:
Maybe we can flex this option for the virtual conf? We won't need to order swag or food, so maybe we can do 1 week (applies to both confs)

Reply:
I am ambivalent about shortening it.